src/app/types/index.ts

```diff
@@ -0,0 +1,24 @@
+export enum Difficulty {
```

Comment:
Change file structure. 

Remove interface folder and types
Add models folder and add this file there and interfaces if there are any. 

Rename this file name to eg. gameSettings.ts or just settings.ts

Reply:
But Difficulty is not used only in settings, so I think it should be in some general file.

Reply:
Okay, we should move it to models folder anyway